controller/PrivateController.php

```diff
@@ -66,11 +70,16 @@ public function getInscription(UserManager $userManager)
         ];
 
         try {
-            $userManager->addUser($datas);
+            $unavailablePseudo = $userManager->getUser($datas['pseudo']);
+            if ($unavailablePseudo === 'false') {
```

Comment:
Are you sure this method will return the string `false` if a user is found with the requested pseudo?
Seems to me you should check if `$unavailablePseudo` is an `instanceof User`

Reply:
You're right. I modify the code. Should be ok now...
Thanks for your help.